system-addon/lib/HighlightsFeed.jsm

```diff
@@ -86,8 +89,7 @@ this.HighlightsFeed = class HighlightsFeed {
 
     // Request more than the expected length to allow for items being removed by
     // deduping against Top Sites or multiple history from the same domain, etc.
-    // Until bug 1425496 lands, do not include saved Pocket items in highlights
-    const manyPages = await this.linksCache.request({numItems: MANY_EXTRA_LENGTH, excludePocket: true});
+    const manyPages = await this.linksCache.request({numItems: MANY_EXTRA_LENGTH, excludePocket: options.excludePocket});
```

Comment:
Mmm… Doesn't this mean we only include pocket items for the one refresh out of the ~12 each hour? So most highlight updates result in no pocket items shown?

Reply:
Other way around, no? We want to update pocket on every refresh except the system_tick unless it's been over an hour... so if options.excludePocket is null for all refreshes, then !excludePocket should be true and then we'd go fetch them in NewTabUtils